CMakeLists.txt

```diff
@@ -172,6 +191,10 @@ function(trusted_firmware_build)
     PUBLIC
     zephyr_interface
     INTERFACE
+    ${PSA_TEST_VAL_FILE}
+    ${PSA_TEST_PAL_FILE}
+    ${PSA_TEST_COMBINE_FILE}
+    ${PLATFORM_NS_FILE}
```

Comment:
Are those really needed on the `zephyr_interface` lib itself ?

Who needs to link with those libs.